model/riscv_insts_zicsr.sail

```diff
@@ -50,13 +51,24 @@ function clause execute CSR(csr, rs1, rd, is_imm, op) = {
   }
 }
 
+function clause execute CSRReg(csr, rs1, rd, op) =
+  doCSR(csr, X(rs1), rd, op, (op == CSRRW) | (rs1 != zreg))
+
+function clause execute CSRImm(csr, imm, rd, op) =
+  doCSR(csr, zero_extend(imm), rd, op, (op == CSRRW) | (imm != zeros_implicit()))
+
+mapping maybe_i : bool <-> string = {
+  true  <-> "i",
+  false <-> ""
+}
```

Comment:
is this used anywhere? I don't see it in this commit

Reply:
No, indeed, looks stale.  I'll remove it.